docker-run.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+# Shell script invoked by next-build docker container (see docker/Dockerfile).
+# Env vars must be present to populate cmd params.
+# Secrets (remaining required params) must be present at ./envs/.env.${ENV}.
+
+APP_ENV=${ENV} yarn dev \
+    --SITE_URL ${SITE_URL} \
+    --NEXT_PUBLIC_DRUPAL_BASE_URL ${NEXT_PUBLIC_DRUPAL_BASE_URL} \
+    --NEXT_IMAGE_DOMAIN ${NEXT_IMAGE_DOMAIN} \
+    --REDIS_URL ${REDIS_URL} \
+    --NEXT_PUBLIC_ASSETS_URL=./public/generated/
```

Comment:
with the way the app is set up, the `APP_ENV` value ties it to a specific .env file. e.g. `APP_ENV=prod` will load env vars from the `envs/.env.prod` file. Do we need to overwrite env vars that would be found in those files?

Reply:
Right, my thought is that k8s supplies env-specific configs in two ways:
1. ConfigMap that sets all the vars found in `docker-run.sh`.
2. Then a k8s Secret creates the correct env file for the given environment, which does contain the remaining configs that happen to be actual secrets (eg. DRUPAL_PREVIEW_SECRET, etc).

Reply:
https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16857#issuecomment-1921388478

Reply:
ah, got it. this is to keep a single image for all the different environments, i missed that note. makes sense 😄